libs/pageserver_api/src/models.rs

```diff
@@ -688,6 +691,9 @@ pub struct TenantConfig {
 
     #[serde(skip_serializing_if = "Option::is_none")]
     pub gc_compaction_ratio_percent: Option<u64>,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub sampling_ratio: Option<Ratio>,
```

Comment:
This needs to be `Option<Option<Ratio>>` if you want the ability to disable perf tracing for a tenant while having it enabled in the default tenant conf.

We might not need that in the end, but, let's please stick to the convention that all `TenantConfig` fields mechanically wrap the config toml field's type.
(Some recently added fields slipped my review)

Reply:
I don't really want that ability, but the convention is fair. Fixed in https://github.com/neondatabase/neon/pull/11433/commits/75dd604e9dbe255775fcd38cc723211dfe51e0e6